package.json

```diff
@@ -27,6 +27,7 @@
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "react-hooks-testing-library": "^0.6.0",
+    "react-test-renderer": "^16.8.6",
```

Comment:
are we using this somewhere?

Reply:
when `useCallback` was in play, I was using `react-hooks-testing-library` and `react-test-renderer` is a peer dependency.  I can remove it but you would need to re-add it later if you are testing certain aspects of hooks.

I'll remove it

Reply:
nah, I just had to add it to my branch. It was complaining.